source/admin_manual/list_of_events.rst

```diff
@@ -27,6 +27,10 @@ Categories
 +----------------+------------------------------------------------------------+
 | metacache      | Metacache library                                          |
 +----------------+------------------------------------------------------------+
+| mail-index     | dovecot.index* file handling events                        |
```

Comment:
you should quote this and the dovecot.index.cache below to avoid accidents.

Reply:
You mean \\*?

Reply:
Or I guess it could be with \`\` also.

Reply:
Looks like another pull request included this change, so this change is already merged. I moved the fixups to https://github.com/dovecot/documentation/pull/67